packages/mongodb-constants/src/atlas-search-templates.ts

```diff
@@ -107,7 +107,9 @@ export const ATLAS_SEARCH_TEMPLATES: SearchTemplate[] = [
     "fields": {
       \${1:<vector embedding field name>}: [
         {
-          "type": "vectorEmbedding"
+          "type": "knnVector",
+          "dimensions": \${2:<number of dimensions>},
+          "similarity": \${3:<euclidean | cosine | dotProduct>}
```

Comment:
Would it make sense for the value for `similarity` be wrapped in a string? I'm wondering if that makes it easier for folks to use.
```suggestion
          "similarity": "\${3:<euclidean | cosine | dotProduct>}"
```